src/test/java/ru/training/at/hw2/ex1/FirstExerciseTest.java

```diff
@@ -0,0 +1,79 @@
+package ru.training.at.hw2.ex1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import ru.training.at.hw2.TestsInitialization;
+
+public class FirstExerciseTest extends TestsInitialization {
+
+    @Test
+    public void exerciseTest() {
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        wait.until(ExpectedConditions.elementToBeClickable(By.id("user-icon")));
+        Assert.assertEquals(driver.getTitle(), "Home Page");
```

Comment:
In first exercise you should use SoftAssertion. Also let's try to separate all expected values in one class, maybe with static variables. It'll help in next HW, but now it's not required.